adafruit_as726x.py

```diff
@@ -316,79 +325,79 @@ def read_calibrated_value(self, channel):
         return struct.unpack("!f", val)[0]
 
     @property
-    def data_ready(self):
+    def data_ready(self) -> bool:
         """True if the sensor has data ready to be read, False otherwise"""
         return (self._virtual_read(_AS726X_CONTROL_SETUP) >> 1) & 0x01
 
     @property
-    def temperature(self):
+    def temperature(self) -> int:
         """The temperature of the device in Celsius"""
         return self._virtual_read(_AS726X_DEVICE_TEMP)
 
     @property
-    def violet(self):
+    def violet(self) -> float:
         """Calibrated violet (450nm) value"""
         return self.read_calibrated_value(_AS7262_VIOLET_CALIBRATED)
 
     @property
-    def blue(self):
+    def blue(self) -> float:
         """Calibrated blue (500nm) value"""
         return self.read_calibrated_value(_AS7262_BLUE_CALIBRATED)
 
     @property
-    def green(self):
+    def green(self) -> float:
         """Calibrated green (550nm) value"""
         return self.read_calibrated_value(_AS7262_GREEN_CALIBRATED)
 
     @property
-    def yellow(self):
+    def yellow(self) -> float:
         """Calibrated yellow (570nm) value"""
         return self.read_calibrated_value(_AS7262_YELLOW_CALIBRATED)
 
     @property
-    def orange(self):
+    def orange(self) -> float:
         """Calibrated orange (600nm) value"""
         return self.read_calibrated_value(_AS7262_ORANGE_CALIBRATED)
 
     @property
-    def red(self):
+    def red(self) -> float:
         """Calibrated red (650nm) value"""
         return self.read_calibrated_value(_AS7262_RED_CALIBRATED)
 
     @property
-    def raw_violet(self):
+    def raw_violet(self) -> int:
         """Raw violet (450nm) 16-bit value"""
         return self.read_channel(_AS7262_VIOLET)
 
     @property
-    def raw_blue(self):
+    def raw_blue(self) -> int:
         """Raw blue (500nm) 16-bit value"""
         return self.read_channel(_AS7262_BLUE)
 
     @property
-    def raw_green(self):
+    def raw_green(self) -> int:
         """Raw green (550nm) 16-bit value"""
         return self.read_channel(_AS7262_GREEN)
 
     @property
-    def raw_yellow(self):
+    def raw_yellow(self) -> int:
         """Raw yellow (570nm) 16-bit value"""
         return self.read_channel(_AS7262_YELLOW)
 
     @property
-    def raw_orange(self):
+    def raw_orange(self) -> int:
         """Raw orange (600nm) 16-bit value"""
         return self.read_channel(_AS7262_ORANGE)
 
     @property
-    def raw_red(self):
+    def raw_red(self) -> int:
         """Raw red (650nm) 16-bit value"""
         return self.read_channel(_AS7262_RED)
 
-    def _virtual_read(self, addr):
+    def _virtual_read(self, addr: int) -> Union[int, float]:
```

Comment:
Did some digging, seems like just `float` should be enough per PEP 484:  https://www.python.org/dev/peps/pep-0484/#the-numeric-tower

Reply:
Huh. That seems less clear to my brain than using `Union`, but PEP 484 is the boss... :-) Committing the change as soon as I respond to these comments. 

Thanks for all your help reviewing this! I've learned a lot about a subject I thought I understood, which is *always* a good thing in my view.